### ISSUE TITLE
[Bug] Terra throwing exception in multivers generator listing

### ISSUE THREAD

**Issue by DemosKadi** [PolyhedralDev/Terra#459]
## Pre-Issue Checklist

- [x] I have checked that I am on the latest version of Terra.
- [x] I have searched the github issue tracker for similar issues, including
  closed ones.
- [x] I have made sure that this is not a bug with another mod or plugin, and it
  is Terra that is causing the issue.
- [x] I have checked that this is an issue with Terra and not an issue with the
  pack I am using.
  
- [x] I have attached a copy of the `latest.log` file [latest.log](https://github.com/user-attachments/files/15539565/latest.log)
- [x] I have filled out and provided all the appropriate information.

## Environment

| Name                         | Value                                                                                                                                                                       |
|------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Terra Version                | 6.4.3-BETA-Bukkit                                                                                                              
| Platform / Platform Version  | paper-1.20.4-496 
| Any External Plugins or Mods | ProtocolLib, vane-{admin,bedtime,core,enchantments,permissions,portals,regions,trifles}, multiverse-{core,inventories,nehterportals,portals,signportals}, quarrycraft-updated, TerraformGenerator, Terra                                       
| Terra Packs In Use           | Origen (also tried SkyLands but not with these logs)     
| Terra Addons In Use          | (no additionall addons installed)   

## Issue Description
When Terra is installed the `mv gens` command only prints the stack trace

### Steps to reproduce

1. Install multiverse plugin
2. Install Terra
3. Run `/mv gens`

### Expected behavior

A list of all world generators, including Terra

### Actual behavior

The list does not get printed, only the stacktrace

### Full stacktrace

<details>
<summary>Stack Trace</summary>

```
[22:59:11 WARN]: java.lang.IllegalArgumentException: No such config pack ""
[22:59:11 WARN]:        at Terra-bukkit-6.4.3-BETA+ab60f14ff-shaded.jar//com.dfsek.terra.bukkit.TerraBukkitPlugin.lambda$getDefaultWorldGenerator$2(TerraBukkitPlugin.java:194)
[22:59:11 WARN]:        at java.base/java.util.Optional.orElseThrow(Unknown Source)
[22:59:11 WARN]:        at Terra-bukkit-6.4.3-BETA+ab60f14ff-shaded.jar//com.dfsek.terra.bukkit.TerraBukkitPlugin.lambda$getDefaultWorldGenerator$3(TerraBukkitPlugin.java:193)
[22:59:11 WARN]:        at java.base/java.util.HashMap.computeIfAbsent(Unknown Source)
[22:59:11 WARN]:        at Terra-bukkit-6.4.3-BETA+ab60f14ff-shaded.jar//com.dfsek.terra.bukkit.TerraBukkitPlugin.getDefaultWorldGenerator(TerraBukkitPlugin.java:192)
[22:59:11 WARN]:        at multiverse-core-4.3.12.jar//com.onarandombox.MultiverseCore.commands.GeneratorCommand.runCommand(GeneratorCommand.java:44)
[22:59:11 WARN]:        at multiverse-core-4.3.12.jar//com.onarandombox.commandhandler.CommandHandler.checkAndRunCommand(CommandHandler.java:296)
[22:59:11 WARN]:        at multiverse-core-4.3.12.jar//com.onarandombox.commandhandler.CommandHandler.processFoundCommands(CommandHandler.java:143)
[22:59:11 WARN]:        at multiverse-core-4.3.12.jar//com.onarandombox.commandhandler.CommandHandler.locateAndRunCommand(CommandHandler.java:93)
[22:59:11 WARN]:        at multiverse-core-4.3.12.jar//com.onarandombox.MultiverseCore.MultiverseCore.onCommand(MultiverseCore.java:839)
[22:59:11 WARN]:        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45)
[22:59:11 WARN]:        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:155)
[22:59:11 WARN]:        at org.bukkit.craftbukkit.v1_20_R3.CraftServer.dispatchCommand(CraftServer.java:999)
[22:59:11 WARN]:        at org.bukkit.craftbukkit.v1_20_R3.CraftServer.dispatchServerCommand(CraftServer.java:984)
[22:59:11 WARN]:        at net.minecraft.server.dedicated.DedicatedServer.bk(DedicatedServer.java:501)
[22:59:11 WARN]:        at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:448)
[22:59:11 WARN]:        at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1525)
[22:59:11 WARN]:        at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1226)
[22:59:11 WARN]:        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319)
[22:59:11 WARN]:        at java.base/java.lang.Thread.run(Unknown Source)
```

</details>

**Comment by DemosKadi** [comment]
Thanks for the info, I will fix the title. Should this issue be closed, since #46 is alreqdy there?

**Comment by solonovamax** [comment]
closing because duplicate, as mentioned
